src/preFilter.ts

```diff
@@ -119,7 +119,8 @@ export function filterByContent(
 		ARR_DIR_REGEX.test(basename(searchee.path)) &&
 		!(
 			searchee.files.length > 1 &&
-			SONARR_SUBFOLDERS_REGEX.test(basename(searchee.path))
+			SONARR_SUBFOLDERS_REGEX.test(basename(searchee.path)) &&
+			nonVideoSizeRatio < fuzzySizeThreshold
```

Comment:
What non video can we reasonably expect to see in arr dirs? Just subtitles? If so we should hardcode to .02 though it doesn't matter.

If we can expect bigger files then we should hardcode it higher. The goal should be to prune what we can expect and leaving rare occurrences to be searched.

Reply:
It would mostly be metadata and subs. Poorly named books, music, and other things are currently being detected.

I think .02 or .05 really doesn't make a big difference in accuracy, but I'd rather be lenient on this identification since its extremely lenient in what qualifies.

What do you think?

Reply:
Lenient in which direction? I think if 99% of media folders have less than 0.02, we should use that.

Reply:
Lenient as in we shouldn't be excluding them aggressively. .02 seems fine. I'll update it.